src/actions/pubSubHandlers.js

```diff
@@ -173,22 +164,51 @@ export function handleLeftChannel({user: userId, channel: channelId}) {
     const users = usersSelector(getState())
     const currentUser = userSelector(getState())
     const isCurrentUser = currentUser.id === userId
-    const user = isCurrentUser ? currentUser : find(users, ({id}) => id === userId)
-
-    dispatch({
-      type: types.REMOVE_USER_FROM_CHANNEL,
-      payload: {
-        channelId,
-        user,
-        isCurrentUser
-      }
-    })
+    const user = isCurrentUser ? currentUser : find(users, ({partner}) => partner.id === userId)
```

Comment:
there should not be such a thing as full list of users on the client.

Reply:
@AleshaOleg as far as I understand there is a list of users (not all of them) that is basically cached in the client. Was this the idea here? If so can you add a comment to the selector that this list necessarily doesn't include all the users of a particular organisation.

Reply:
@nikgraf That's what I did here: https://github.com/ubergrape/grape-web-client/blob/release/src/selectors/index.js#L17

Maybe it's better to post this comment somewhere else?